app/utils/profileManagement.js

```diff
@@ -109,6 +124,22 @@ export async function saveProfile(ethereumService, user, profile) {
   ]);
 }
 
+export async function getUserRatings(ethereumService, user, communities) {
+  if (!communities || !user) {
+    return [];
+  }
+  const ratings = [];
+  await Promise.all(
```

Comment:
why not just use for await construction, with for await you can remove Promise.all

Reply:
I like Promise.all more because of better error handling

Reply:
as I see it's redundant, because you are not handling error cases where you are calling this function, and this construction adds more complexity,
p